src/components/MultiSelect/MultiSelect.js

```diff
@@ -207,11 +208,9 @@ export default class MultiSelect extends React.Component {
                       const itemProps = getItemProps({ item });
                       const itemText = itemToString(item);
                       const isChecked =
-                        selectedItem
-                          .map(selected => {
-                            return selected.id;
-                          })
-                          .indexOf(item.id) !== -1;
+                        selectedItem.findIndex(selected => {
```

Comment:
`.findIndex()` is not supported by IE11 and we refrain from usage of new polyfills given it tends to be a breaking change. Could you try an implementation without `.findIndex()`?

Reply:
Okay good to know 👍 that answers a question I had. I will try one without it